linera-execution/src/evm/database.rs

```diff
@@ -378,4 +379,35 @@ where
             blob_excess_gas_and_price,
         })
     }
+
+    pub fn constructor_argument(&self) -> Result<Vec<u8>, ExecutionError> {
+        let mut runtime = self.runtime.lock().expect("The lock should be possible");
+        let constructor_argument = runtime.application_parameters()?;
```

Comment:
So application _parameters_ are treated differently between EVM and Linera VM? i.e. for the latter they are application-specific parameters that are different from a constructor?

Reply:
The application parameters are treated the same way in Wasm and Evm:
* They are serialized with `serde_json`.
* For the EVM, this is somewhat cumbersome since the input is `Vec<u8>`, but that is how it is.